unified-to/unified-angular-integrations-directory
# Bug/Update Angular library

## What's fixed

- addressed type errors
- extracted list filtering behaviour into a pipe
- separated ngIf and ngFor (cannot be used on same directive)
- use correct syntax for `@Input() workspace_id`
- renamed the component from `UnifiedAngularDirectoryComponent` to `UnifiedDirectory` because that is what our users will be importing and referencing and to be consistent with our other libraries
- the example in the README was wrong - in Angular you use selectors which are written in kebab case. Also, the selector for the component was misnamed i.e. it was `unified` when it should be `unified-directory`
	- also update README to show import instructions

Library build issues:
- we need to publish the library from `dist/unified-angular-directory` NOT the entire workspace i.e. root
- therefore, if we want this library to be called `@unified-api/angular-directory` then:
	- in projects/unified-angular-directory/package.json, rename it   `"name": "@unified-api/angular-directory",`
	- in `/package.json` (the root), give it another name

## How to test

- `cd dist/unified-angular-directory`
- `npm link`
- cd back to root
- run `ng build --watch`
- create a new Angular project
- run `npm link @unified-api/angular-directory`
- the package is now available in the project (it doesn't appear in package.json but should appear under node_modules)

![Screenshot 2024-10-17 at 10 22 56 AM](https://github.com/user-attachments/assets/7d4e46a1-62ea-4837-83a8-5206de03fa95)
